core/parachain/validator/backing_implicit_view.hpp

```diff
@@ -46,6 +46,15 @@ namespace kagome::parachain {
       return r;
     }
 
+    void printStoragesLoad() {
+      SL_TRACE(logger,
+               "[Backing implicit view statistics]:"
+               "\n\t-> leaves={}"
```

Comment:
Please note that this will omit the standard log line prefix with timestamp, thread name, etc.